aep/general/0124/aep.md.j2

```diff
@@ -78,8 +77,54 @@ recommended if additional metadata is required in the relationship, or if the
 restrictions around the use of a repeated field preclude the use of that
 approach.
 
+### Embedded resources
```

Comment:
I think this is missing guidance that the resource field must be output-only? retrieval is one thing, but mutation would be a whole can of worms that leads to conflicts for declarative clients.

Reply:
Yes, agreed.  I realized that (and said as much) when responding to another one of your comments.  Updated.